src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -123,6 +131,8 @@
             this::delete,
             "Takes the specified parameters and calls CatalogFramework::query. `params` takes:"
                 + " `ids` (Required, value: List(String))"));
+
+    builder.put("ddf.catalog/clone", new DocMethod(this::clone, ""));
```

Comment:
✏️ Could add documentation for the params this takes, similar to the delete call above.